sklearn/tests/test_naive_bayes.py

```diff
@@ -647,15 +616,6 @@ def test_cnb():
     assert_array_almost_equal(clf.feature_log_prob_, normed_weights)
 
 
-def test_naive_bayes_scale_invariance():
-    # Scaling the data should not change the prediction results
```

Comment:
Moved up to other gnb tests and renamed to `test_gnb_naive_bayes_scale_invariance`